### PR TITLE
Fix Ergodox KEYMAP macros

### PR DESCRIPTION
**Pull request by drashna**
To use the LAYOUT convention by default, and then map KEYMAP to that for compatibility. 

I think @skullydazed mentioned some fix to the Configurator that may fix this.  But just in case.... here is this PR

### CODE REVIEW

**Comment by drashna** [comment]
Travis CI errors are unrelated

File: keyboards/ergodox_infinity/ergodox_infinity.h [hunk 1]
```diff
@@ -151,7 +149,7 @@ inline void ergodox_led_all_set(uint8_t n) {
     { B80,   B81,   B82,   B83,   B84 }  \
 }
 
-#define LAYOUT_ergodox KEYMAP
-#define LAYOUT_ergodox_pretty KEYMAP_PRETTY
+#define KEYMAP LAYOUT_ergodox
```
Comment (mechmerlin) [inline keyboards/ergodox_infinity/ergodox_infinity.h]: Why do we need to #define KEYMAP to this?

Comment (drashna) [reply keyboards/ergodox_infinity/ergodox_infinity.h]: We don't, this was a "quick and dirty" change, and I didn't want to change all of the keymaps.

**Comment by drashna** [comment]
Fixed in #3171

**Pull request closed without merging**